setup.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import re
+from setuptools import setup
+import setuptools
+
+with open("README.md", "r") as doc:
+    long_description = doc.read()
+
+
+version = re.search(
+    r'^__version__\s*=\s*"(.*)"',
+    open("AutoFlow_OmicsDataHandling/__init__.py").read(),
+    re.M,
+).group(1)
+
+
+setup(
+    name="AutoFlow_OmicsDataHandling",
+    version=version,
+    author="AutoFlow",
+    author_email="TBD",
+    description=(
+        "General Repository for Omics Data Handling tools"
+    ),
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Biosustain/AutoFlow-OmicsDataHandling",
```

Comment:
This URL looks out of date.  The name `AutoFlow_OmicsDataHandling` also appears out of date.

Reply:
I left two things commented in there that we're not using right now but maybe later (console entry point and dependency links)